src/quicklist.h

```diff
@@ -56,13 +56,14 @@ typedef struct quicklistNode {
     unsigned int extra : 10; /* more bits to steal for future usage */
 } quicklistNode;
 
-/* quicklistLZF is a 4+N byte struct holding 'sz' followed by 'compressed'.
+/* quicklistLZF is a 8+N byte struct holding 'sz' followed by 'compressed'.
  * 'sz' is byte length of 'compressed' field.
  * 'compressed' is LZF data with total (compressed) length 'sz'
  * NOTE: uncompressed length is stored in quicklistNode->sz.
- * When quicklistNode->zl is compressed, node->zl points to a quicklistLZF */
+ * When quicklistNode->zl is compressed, node->zl points to a quicklistLZF
```

Comment:
Should this comment line be deleted?

Reply:
what's wrong with that comment? (other than the reference to `zl` which we renamed?

Reply:
@oranagra This comment has been changed to ` * When quicklistNode->entry is compressed, node->entry points to a quicklistLZF */` and old has not been deleted.